ui-components/src/actions/ButtonGroup.scss

```diff
@@ -0,0 +1,61 @@
+.button-group {
+  --background-color: var(--bloom-color-primary-lighter);
+  --padding-block: var(--bloom-s8);
+  --padding-inline-desktop: var(--bloom-s16);
+  --padding-inline-mobile: var(--bloom-s4);
+  --inner-button-gap: var(--bloom-s3);
+
+  --padding-inline: var(--padding-inline-desktop);
+
+  @media (max-width: $screen-sm) {
+    --padding-inline: var(--padding-inline-mobile);
+  }
+
+  background-color: var(--background-color);
+  padding-block: var(--padding-block);
+  padding-inline: var(--padding-inline);
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--inner-button-gap);
+
+  &.has-between-spacing {
+    justify-content: space-between;
+  }
+
+  &.has-even-spacing {
+    justify-content: space-evenly;
+  }
+}
+
+.button-group__column {
+  display: flex;
+  gap: var(--inner-button-gap);
+
+  .button {
+    @media (max-width: $screen-sm) {
+      --normal-rounded: var(--small-rounded);
+      --normal-padding: var(--small-padding);
+      --normal-font-size: var(--small-font-size);
+    }
+  }
+}
+
+.button-group.has-fullwidth-mobile-buttons {
```

Comment:
This is more a general question. Are we following a standard of any sort to decide when to write out the full path like it is here or when we just put the second chained class within the block like how it is done above for `.has-between-spaces`? I'm indifferent on which one, but we might want to do the same throughout the file

Reply:
Good point, I worked on a better nesting structure for the parent/child relationship and the variants.